pandas/tests/extension/decimal/test_decimal.py

```diff
@@ -86,7 +86,7 @@ def convert(x):
         else:
             right_na = right.isna()
 
-        tm.assert_series_equal(left_na, right_na)
+        tm.assert_series_equal(left_na, right_na, *args, **kwargs)
```

Comment:
why are you changing this?

Reply:
It's a bug I bumped into while passing `check_index_type=False` to the tests in an earlier version of this PR. With the current changes, it might not be related anymore.